genie-core/src/main/java/com/netflix/genie/core/jobs/JobConstants.java

```diff
@@ -135,6 +135,11 @@
     public static final String GENIE_KILL_REASON_FILE_NAME = "genie/kill-reason";
 
     /**
+     * File created by Genie with details and trace for a job that failed to initialize.
+     **/
+    public static final String GENIE_INIT_FAILURE_MESSAGE_FILE_NAME = "init-failure-details";
```

Comment:
we might want to think about making this file a `.txt` or similar so that mime types can read it properly. Also dashes in file name do we think that's best?